packages/arm-core/src/components/DrawerOverlay/DrawerPanelHeader.tsx

```diff
@@ -0,0 +1,41 @@
+import { Close } from '@amsterdam/asc-assets'
+import { Button } from '@amsterdam/asc-ui'
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components'
+
+const HeaderContainer = styled.div`
+  display: flex;
+`
+
+const HeaderContent = styled.div``
+
+const CloseButton = styled(Button)`
+  margin-left: auto;
+`
+
+export interface DrawerPanelHeaderProps {
+  onClose?: () => void
+}
+
+const DrawerPanelHeader: FunctionComponent<DrawerPanelHeaderProps> = ({
+  children,
+  onClose,
+}) => {
+  return (
+    <HeaderContainer>
+      <HeaderContent>{children}</HeaderContent>
```

Comment:
This component is a div without styling. Should you be using a styled component here? A `<div>` would suffice, wouldn't it?

Reply:
Yeah, had some styles there before. Removed it.